utils/log/logger.go

```diff
@@ -40,89 +50,82 @@ type jfrogLogger struct {
 func SetLogger(newLogger Log) {
 	Logger = newLogger
 }
-
 func (logger *jfrogLogger) SetLogLevel(LevelEnum LevelType) {
 	logger.LogLevel = LevelEnum
 }
-
```

Comment:
Let's restore this new-line (we separate functions with a new-line).
This applies to other places in this file.